config/default.yml

```
@@ -725,6 +725,15 @@ Rails/TimeZone:
   Exclude:
     - '**/*.gemspec'
 
+Rails/TimeZoneAssignment:
+  Description: 'Prefer the usage of `Time.use_zone` instead of manually updating `Time.zone` value.'
+  Reference: 'https://thoughtbot.com/blog/its-about-time-zones'
+  Enabled: true
```

Thank you for updating this PR. New cops will start with `pending` status. Can you update the status?

```suggestion
  Enabled: 'pending'
```